packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -185,13 +193,16 @@ export class TokenRatesController extends BaseController<
       onNetworkStateChange: (
         listener: (networkState: NetworkState) => void,
       ) => void;
+      onPreferencesStateChange: (
+        listener: (networkState: PreferencesState) => void,
+      ) => void;
     },
     config?: Partial<TokenRatesConfig>,
     state?: Partial<TokenRatesState>,
   ) {
     super(config, state);
     this.defaultConfig = {
-      disabled: true,
+      disabled: false,
```

Comment:
Is this change of default intentional??

Reply:
In practice, before this PR, it was defaulting to false, because of the `this.configure({ disabled: false }, false, false);` line below. In fact, that line would override the `config.disabled` property passed to the constructor as well, effectively forcing any instantiation of this class to always have a `false` disabled property.